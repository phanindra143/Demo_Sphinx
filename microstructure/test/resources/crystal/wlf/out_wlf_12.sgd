*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   116.1794
  !k_ij(1,2)=     0.2890
  !k_ij(2,2)=   116.4375
  !k_ij(1,3)=     0.6213
  !k_ij(2,3)=     0.2779
  !k_ij(3,3)=   116.1146
MP,Kxx,1,  116.17941
MP,Kyy,1,  116.43754
MP,Kzz,1,  116.11456
MP,Kxx,2,  116.43754
MP,Kyy,2,  116.17941
MP,Kzz,2,  116.11456
MP,Kxx,3,  116.43754
MP,Kyy,3,  116.17941
MP,Kzz,3,  116.11456
MP,Kxx,4,  116.17941
MP,Kyy,4,  116.43754
MP,Kzz,4,  116.11456
MP,Kxx,5,  116.17941
MP,Kyy,5,  116.43754
MP,Kzz,5,  116.11456
MP,Kxx,6,  116.43754
MP,Kyy,6,  116.17941
MP,Kzz,6,  116.11456
MP,Kxx,7,  116.43754
MP,Kyy,7,  116.17941
MP,Kzz,7,  116.11456
MP,Kxx,8,  116.17941
MP,Kyy,8,  116.43754
MP,Kzz,8,  116.11456
MP,Kxx,9,  116.11456
MP,Kyy,9,  116.17941
MP,Kzz,9,  116.43754
MP,Kxx,10,  116.17941
MP,Kyy,10,  116.11456
MP,Kzz,10,  116.43754
MP,Kxx,11,  116.17941
MP,Kyy,11,  116.11456
MP,Kzz,11,  116.43754
MP,Kxx,12,  116.11456
MP,Kyy,12,  116.17941
MP,Kzz,12,  116.43754
MP,Kxx,13,  116.11456
MP,Kyy,13,  116.17941
MP,Kzz,13,  116.43754
MP,Kxx,14,  116.17941
MP,Kyy,14,  116.11456
MP,Kzz,14,  116.43754
MP,Kxx,15,  116.17941
MP,Kyy,15,  116.11456
MP,Kzz,15,  116.43754
MP,Kxx,16,  116.11456
MP,Kyy,16,  116.17941
MP,Kzz,16,  116.43754
MP,Kxx,17,  116.43754
MP,Kyy,17,  116.11456
MP,Kzz,17,  116.17941
MP,Kxx,18,  116.11456
MP,Kyy,18,  116.43754
MP,Kzz,18,  116.17941
MP,Kxx,19,  116.11456
MP,Kyy,19,  116.43754
MP,Kzz,19,  116.17941
MP,Kxx,20,  116.43754
MP,Kyy,20,  116.11456
MP,Kzz,20,  116.17941
MP,Kxx,21,  116.43754
MP,Kyy,21,  116.11456
MP,Kzz,21,  116.17941
MP,Kxx,22,  116.11456
MP,Kyy,22,  116.43754
MP,Kzz,22,  116.17941
MP,Kxx,23,  116.11456
MP,Kyy,23,  116.43754
MP,Kzz,23,  116.17941
MP,Kxx,24,  116.43754
MP,Kyy,24,  116.11456
MP,Kzz,24,  116.17941
*endif
