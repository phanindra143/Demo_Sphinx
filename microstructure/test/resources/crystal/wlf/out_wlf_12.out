            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\3
             3.00000      -1815282.86561         -5407.60088         -5585.10157
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000             5.22820           -35.20227
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         -3746.47656      -1819327.95777         -1177.41149
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000            -3.98009         15625.00000           -25.33580
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         -4914.24701         -6914.87485      -1814269.35749
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -41.62371            20.31108         15625.00000
  /%
